instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/util.py

```diff
@@ -48,25 +48,29 @@ def _extract_conn_attributes(conn_kwargs):
     return attributes
 
 
-def _format_command_args(args):
-    """Format command arguments and trim them as needed"""
-    value_max_len = 100
-    value_too_long_mark = "..."
-    cmd_max_len = 1000
-    length = 0
-    out = []
-    for arg in args:
-        cmd = str(arg)
+def _format_command_args(args, sanitize_query):
+    """Format and sanitize command arguments, and trim them as needed"""
+    if sanitize_query:
+        # Sanitized query format: "COMMAND ? ?"
+        out = [str(args[0])] + ["?"] * (len(args) - 1)
```

Comment:
Will there ever be a case where args[0] raises index error?

Reply:
Not that I know off. I assume index 0 is always the Redis command that's being performed. I can add more safeguards if that helps.

Reply:
That would be good because we don't want to instrumentation to raise exceptions ever. Users would rather lose telemetry than seeing the failures from instrumentation.